cloudinit/sources/DataSourceAzure.py

```diff
@@ -286,6 +286,7 @@ def get_resource_disk_on_freebsd(port_id) -> Optional[str]:
     "data_dir": AGENT_SEED_DIR,
     "disk_aliases": {"ephemeral0": RESOURCE_DISK_PATH},
     "apply_network_config": True,  # Use IMDS published network configuration
+    "apply_ip_network_config": True,  # Set secondary ip addresses
```

Comment:
`apply_network_config_for_secondary_ips` would be clearer

Reply:
Done!